examples/ansible/setup_neuvector.yml

```diff
@@ -0,0 +1,104 @@
+# The playbook supports two operations: setup and removal of NeuVector.
+# The operation is determined by the 'neuvector_state' variable, which can be set to 'present' or 'absent'.
+# When 'neuvector_state' is set to 'present', the playbook will:
+#   - Check if the tools for the NeuVector workload are already installed
+#   - Set SELinux into permissive mode
+#   - Retrieve the NeuVector image
+#   - Execute nevector runlabel INSTALL
+#   - Start the NeuVector service
+#   - Enable the NeuVector service to start on a reboot
+#   - Print a message with the connection details for NeuVector
+# When 'neuvector_state' is set to 'absent', the playbook will:
+#   - Stop the NeuVector service
+#   - Execute nevector runlabel UNINSTALL
+#   - Remove the NeuVector image
+
+- name: Running the NeuVector workload
+  hosts: alphost
+  vars:
+    neuvector_state: present
+    workload:
+      name: neuvector
+      image: registry.opensuse.org/suse/alp/workloads/bci_containerfiles/suse/alp/workloads/neuvector-demo:latest
+
+  tasks:
+    - name: Setup NeuVector
+      when:
+        - neuvector_state == "present"
+      block:
+        - name: Check if workload {{ workload.name }} is already installed
+          become: true
+          containers.podman.podman_container_info:
+            name: "{{ workload.name }}-allinone"
+          register: workload_info
+          changed_when: false
+
+        - name: Set SELinux into permissive mode
+          become: true
+          ansible.posix.selinux:
+            policy: targeted
+            state: permissive
+
+        - name: Retrieve image for workload {{ workload.name }}
+          become: true
+          containers.podman.podman_image:
+            name: "{{ workload.image }}"
+            state: present
+          when:
+            - workload_info.containers | length == 0
+
+        - name: Execute nevector runlabel INSTALL
+          become: true
+          ansible.builtin.command: >-
+            podman container runlabel install {{ workload.image }}
+          register: workload_runlabel_install
+          changed_when:
+            - ('already exist' not in workload_runlabel_install.stdout)
+
+        - name: Start NeuVector service
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            state: started
+          register: workload_service_start
+
+        - name: enable service to start on a reboot
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            enabled: yes
+
+        - name: Print message connect to NeuVector
+          ansible.builtin.debug:
+            msg: >-
+              NeuVector is running on https://{{ ansible_default_ipv4.address }}:8443
+              You need to accept the warning about the self-signed SSL certificate
+              and log in with the following default credentials: admin / admin.
+          when:
+            - workload_service_start is success
```

Comment:
This condition is also not needed as when the service is not successfully started then this will not be executed.